gopem/helper.py

```diff
@@ -116,7 +116,7 @@
 StyleList = sorted(StyleTable)
 StyleList.insert(0, StyleList.pop(StyleList.index("Solid")))
 
-ScaleList = ["Linear", "Log"]
+ScaleList = ["linear", "log"]
```

Comment:
Keeping the new `ScaleList` will affect the UI since all other sections are capitalized, and only this menu will be in lowercase.

Reply:
ok, didn't notice that, will fix it now